src/components/NavBar.astro

```diff
@@ -0,0 +1,54 @@
+<div class="nav-class">
+  <div>
+    <object
+      id="svg1"
+      width="59"
+      height="46"
+      data="/public/code-to-cloud-icon.svg"
+      type="image/svg+xml"></object>
+  </div>
+  <nav>
+    <a href="">Home</a><a href="">Speakers</a><a href="">Agenda</a><a href=""
+      >Values</a
+    ><a href="">Team</a><a href="">Pricing</a><a href="">Blog</a><a href=""
+      >Buy</a
+    >
+  </nav>
+</div>
+
+<style>
+  .nav-class {
+    background-color: #1c2120;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    padding: 17px 25px 17px 25px;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    column-gap: 16px;
+    align-items: center;
+  }
+
+  nav a {
+    padding: 16px 20px 16px 20px;
+    color: #faf9f6;
+    font-weight: 600;
+    font-size: 14px;
+    text-decoration: none;
+    text-align: center;
+    transition: 0.25s;
+  }
+
+  nav a:hover {
+    background-color: #3a3b45;
+    border-radius: 10px;
+  }
+
+  nav a:last-child {
+    border: 1px solid #faf9f6;
+    border-radius: 10px;
+  }
```

Comment:
@lindan4 what are your thoughts on handling CSS. I think since it's a small project I like this approach of keeping it in line. Did you have any extra thoughts?